xtask/Cargo.toml

```diff
@@ -9,5 +9,6 @@ chrono     = "0.4"
 clap       = { version = "4.5", features = ["derive"] }
 env_logger = "0.11"
 log        = "0.4"
+pyo3       = { version = "0.27.1", features = ["auto-initialize"] }
```

Comment:
Can we have this behind a feature? This way we would only need `python3` when regenerating the efuses, but the rest of the `xtask` commands can be run without it. We could add an alias, just like in https://github.com/esp-rs/esp-hal/blob/cd8addbedde320c9352020b2dde142e7e83c2aa7/.cargo/config.toml#L4 so its easier too.

Reply:
Like this?  https://github.com/esp-rs/espflash/pull/971/commits/03f86c80853e3c60702ea479a6913b2dca400791

Reply:
Yeah! Thanks!